Gemfile

```diff
@@ -23,4 +23,4 @@ gem "wdm", "~> 0.1.1" if Gem.win_platform?
 
 gem "html-proofer", '>=5.0.7'
 
-gem "webrick", ">= 2.2.8"
+gem "webrick", '>= 1.8.1'
```

Comment:
Why reduce the version?

Reply:
@b00f latest is 1.8.1

https://rubygems.org/gems/webrick/